librosa/core/audio.py

```diff
@@ -638,6 +641,10 @@ def get_duration(y=None, sr=22050, S=None, n_fft=2048, hop_length=512,
             n_samples = n_samples - 2 * int(n_fft / 2)
 
     else:
+        # Ensure Fortran contiguity.
+        if not np.isfortran(y):
+            y = np.asfortranarray(y)
```

Comment:
This is equivalent to `y = np.asfortranarray(y)`, which returns the input unmodified if it's already F-contig.

```python
In [2]: x = np.ones(5)                                                                        

In [3]: x.flags['F_CONTIGUOUS']                                                               
Out[3]: True

In [4]: y = np.asfortranarray(x)                                                              

In [5]: y is x                                                                                
Out[5]: True
```

Reply:
fixed in d4decceb